lblod/manage-submission-form-tooling
# Adding new LEKP forms and adjusting LEKP form melding correctie authentieke bron

# Description

DL-5498 & DL-5499

This PR : 
- Removes the `Type Correctie` field from LEKP form melding correctie authentieke bron.
- Adds three new forms `LEKP-rapport - Sloopbeleidsplan`, `LEKP-rapport - Collectieve renovatie` and  `LEKP-rapport - Fietspaden`

# Type of change


# Related services

- enrich-submission
- mu-migrations

# How to test 

1. Generate dist file and copy the forms to consuming apps
2. Log in loket to check if the form field has been removed

**LEKP form melding correctie authentieke bron**

![Screenshot 2023-12-05 at 12-07-29 Loket voor lokale besturen](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/33d44b2a-12ff-4f6d-b009-0193bad5119f)

**LEKP-rapport - Sloopbeleidsplan**

![lekp-rapport-sloopbeleidsplan](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/4f4d5f9a-b88b-4c81-8838-d16c3f320058)

Revision : 

![Screenshot 2023-12-15 at 12-54-02 Loket voor lokale besturen](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/acfef1c6-3339-4a7a-bcfa-87135767bfc0)


**LEKP-rapport - Collectieve energiebesparende renovatie**

Old version :

![lekp-rapport-collectieve-renovatie](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/7adfcbd8-31ff-45d8-820e-b33d11a2014c)

Revision : 

![Screenshot 2023-12-15 at 11-45-27 Loket voor lokale besturen](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/1298ccff-1d56-4a7c-bbee-9b44755592ef)


**LEKP-rapport - Fietspaden**

![Fietspaden-form](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/eb6bcb9c-39ce-493b-b9c1-9ef0bc163feb)


4. Save the form, it should be saved without errors.
5. Send the form, it should be consumed by app-toezicht-abb and not by app-public-decisions-database.
a. To consume the form you can follow the docs @ https://github.com/lblod/app-toezicht-abb#trigger-import-export-flow-from-app-digitaal-loket
 
# What to check

- Typos, and form structure  

# Links to other PR's

- https://github.com/lblod/manage-submission-form-tooling/pull/40
- https://github.com/lblod/enrich-submission-service/pull/16
- https://github.com/lblod/app-meldingsplichtige-api/pull/38
- https://github.com/lblod/app-digitaal-loket/pull/502
- https://github.com/lblod/app-public-decisions-database/pull/21
- https://github.com/lblod/app-worship-decisions-database/pull/53
- https://github.com/lblod/app-toezicht-abb/pull/35

# Notes

- drc restart migrations resource cache enrich-submission
